src/api-reference/request/v4.endpoints.request-resources.markdown

```diff
@@ -0,0 +1,755 @@
+---
+title: Travel Request v4 - Endpoints - Request Resources
+layout: reference
+---
+
+{% include prerelease.html %}
+
+If a new or existing project please use version [3.0](./v3.request.html) of this API.
+
+* [Request Resources](#request-resource)
+  * [Create a new Request](#request-resource-create)
+  * [Get the list of existing Requests](#request-resource-getlist)
+  * [Get the content of an existing Request](#request-resource-get)
+  * [Update the content of an existing Request](#request-resource-update)
+  * [Delete an existing Request](#request-resource-delete)
+  * [Create an expense report linked to an approved Request](#request-resource-create-expense-linked)
+
+## HTTP Status Codes
+
+To learn more about response HTTP status codes for this API see [Travel Request v4 - HTTP Status Codes](./v4.response-codes.html).
+
+# <a name="request-resource"></a>Request Resources
+
+Manage documents used for pre-spend authorizations within Concur Request
+
+## <a name="request-resource-create"></a>Create a new Request
+
+### Scopes
+
+`travelrequest.write` - Refer to [Scope Usage](./v4.get-started.html#scope-usage) for full details.
+
+### HTTP Request
+
+#### Parameters
+
+Name|Parameter Type|Data Type|Description
+---|-------|--------|------
+userId|path|string|The unique identifier of the Request owner for whom the Request will be created. <br> The corresponding user name will be displayed in the audit trail of the Request <br> **Required** when connecting with a Company token, if empty a 400 `missingRequiredParam` error code will be displayed
+
+
+##### URI Template
+
+```
+POST /v4/requests
+```
+
+#### Payload
+
+[Request](./v4.endpoints.schemas.html#schema-request)
+
+### HTTP Response
+
+#### Payload
+
+[Request](./v4.endpoints.schemas.html#schema-request) - The created Request
+
+### Example
+
+#### HTTP Request
+
```

Comment:
This example request has a URL of datacenter/travelrequest/v4/requests, when the response has a URL of datacenter/v4/requests. Please contact the PM and ask which URL is correct. Then check all the examples, these URLs are different throughout.